packages/m2-theme/public/index.php

```diff
@@ -17,7 +18,8 @@
     <link rel="preload" href="https://use.typekit.net/fji5tuz.css" as="style"> -->
 
     <meta name="theme-color" content="#<?= $themeColor ?: 'ffffff' ?>">
-
+    <?= $head_scripts ?>
```

Comment:
what is the value in case it's not set? maybe it would be better to have an if check before and only then render it?

Reply:
No need for if check, `getThemeConfiguration` returns `null` which will output nothing into page.